module/Kanbanize/src/Kanbanize/Controller/Console/KanbanizeToOraSyncController.php

```diff
@@ -284,4 +308,83 @@ private function updateTaskPositionFromKanbanize($task, $kanbanizeTask, $systemU
             $this->transaction()->rollback();
         }
     }
+
+    public function getLanesSyncReport($kanbanizeFullStructure, $org)
```

Comment:
questo metodo ha parecchie responsabilità diverse, capire cosa fa non è immediato.. non so quanto tempo è stato investito nel task, ma potrebbe valere la pena gestire le lane come è stato fatto per i settings, usando un oggetto (Lanes? LaneCollection?) che prende in pancia l'array delle lane prese da kanbanize e decide cosa fare (update, delete, add)